Business chatbox UI

jmyanASU opened the pull request:
Business chatbox window, Any advice would be greatly appreciated. Thanks!
<img width="1071" alt="Screen Shot _UI" src="https://github.com/NikitaSalikov/Capstone/assets/144946572/99751059-17fc-49fc-9e39-dff237dcdcd5">

jmyanASU committed:
Business chatbox UI

Business chatbox window

NikitaSalikov reviewed (commented):
This looks like a good start!

about the code: I assume this is a mock-up html? This is all hardcoded, is it possible to implement the clicking and transferring to other chats functionality so that the chat with that person/business appears? It will require using the scripts. Also, don't forget that we're going to use React.JS!

Other than that - good job!

P.S. I am not going to approve yet - Ghaida might need to look at the code

jmyanASU committed:
Updated Business UI to React

Buttons are not working, need to be implemented, and replaced by react icon lib.
Will work on React States to add more interaction over the weekend.

NikitaSalikov committed:
Merge pull request #14 from NikitaSalikov/BusinessUI_React

Updated Business UI to React
